sn_cli/src/subcommands/files/mod.rs

```diff
@@ -251,6 +258,29 @@ pub(crate) async fn files_cmds(
     Ok(())
 }
 
+pub(crate) async fn estimate_cost(path: PathBuf, client: &Client, root_dir: &Path) -> Result<()> {
+    let mut chunk_manager = ChunkManager::new(&root_dir);
+    chunk_manager.chunk_path(&path, true, true)?;
+
+    let files_api: FilesApi = FilesApi::new(client.clone(), root_dir.to_path_buf());
+    let mut total: u64 = 0;
+
+    for chunk_detail in chunk_manager.get_chunks() {
+        let chunk_address = ChunkAddress::new(chunk_detail.0);
+        let address = NetworkAddress::from_chunk_address(chunk_address);
+        let estimate = files_api
+            .wallet()?
+            .get_store_cost_at_address(address)
+            .await
+            .unwrap()
```

Comment:
We don't use unwraps in prod code, please. Use the try operator `?` and advance appropriate errors upwards (or `bail` via `eyre` with context)